[FV sguuxs] New mobile and desktop Sgüüx̱s keyboards

HopsAndHops opened the pull request:
Fully new build.

keyman-server commented:
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

HopsAndHops commented:
Added in Aboriginal Serif and Aboriginal Sans and removed the unnecessary files. 

We're leaving the OSKs for now - if we have time towards the end of the project we may return and remove them across the board.

keyman-server commented:
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

LornaSIL commented:
Somehow `fv_haisla.kpj` crept into this PR too! Can you remove that?

keyman-server commented:
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

LornaSIL reviewed (approved):
LGTM

darcywong00 commented on release/fv/fv_sguuxs/source/fv_sguuxs.kps:
```diff
@@ -0,0 +1,103 @@
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>..\..\..\..\..\..\..\Downloads\sguuxs layout images\fv_sguuxsU_.png</Name>
```
The CI build failed because this file doesn't exist for including in the keyboard package.
```bash
fv_sguuxs.kps: Compiling package fv_sguuxs.kps...
22:25:58   fv_sguuxs.kps: Warning: File C:\BuildAgent\work\aee9ab97cd908689\keyboards\release\fv\fv_sguuxs\source\..\..\..\..\..\..\..\Downloads\sguuxs layout images\fv_sguuxsU_.png does not exist.
22:25:58   fv_sguuxs.kps: Error: The build was not successful. Some files were skipped.
22:25:58   fv_sguuxs.kps: Failure: 'C:\BuildAgent\work\aee9ab97cd908689\keyboards\release\fv\fv_sguuxs\source\fv_sguuxs.kps' was not compiled successfully.
```

```suggestion
      <Name>welcome\fv_sguuxsU_.png</Name>
```

darcywong00 commented:
At some point after this is released, in order to get this keyboard distributed in the mobile apps, one of us will needed to update:
* fv_all kps version in the [source file](https://github.com/keymanapp/keyboards/blob/02603516ed2382d610d5571a0cd2dbdd3fa4d416/release/packages/fv_all/source/fv_all.kps.in#L21)
* add fv_sguux entry (for BC Coast region / tsi - Tsimshian language) in [keyboards.csv](https://github.com/keymanapp/keyman/blob/master/oem/firstvoices/keyboards.csv)

HopsAndHops commented:
oof, sorry about that. I'll keep a closer eye on that going forward, seems that quite a few of the keyboards I've been sitting on from earlier this year have this mistake.

keyman-server commented:
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

darcywong00 reviewed (approved):
lgtm
